Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -240,9 +240,8 @@ class MultiTrajectory {
   /// @param iprevious index of the previous state, kInvalid if first
   /// @return Index of the newly added track state
   template <bool RO = ReadOnly, typename = std::enable_if_t<!RO>>
-  constexpr IndexType addTrackState(
-      TrackStatePropMask mask = TrackStatePropMask::All,
-      IndexType iprevious = kInvalid) {
+  IndexType addTrackState(TrackStatePropMask mask = TrackStatePropMask::All,
+                          IndexType iprevious = kInvalid) {
```

Comment:
I wonder if this ever really was constexpr because the underlying vector impl will not be constexpr for example?

This is one of the changes I would factor out

Reply:
I moved this to https://github.com/acts-project/acts/pull/3073

We can clean this PR out as soon as the others are merged. Or you can do it already if you like!